flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -482,6 +483,23 @@ public void testCassandraBatchFormats() throws Exception {
 		Assert.assertEquals(20, result.size());
 	}
 
+	@Test
+	public void testCassandraBatchRowFormats() throws Exception {
```

Comment:
rename to `testCassandraBatchRowFormat` (-s)

Reply:
done